ci/test_wheel_cuproj.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+set -eou pipefail
+
+mkdir -p ./dist
+RAPIDS_PY_CUDA_SUFFIX="$(rapids-wheel-ctk-name-gen ${RAPIDS_CUDA_VERSION})"
+RAPIDS_PY_WHEEL_NAME="cuproj_${RAPIDS_PY_CUDA_SUFFIX}" rapids-download-wheels-from-s3 ./dist
+
+# Install additional dependencies
+apt update
+DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends libgdal-dev
```

Comment:
Does cuproj need libgdal?

Reply:
IIRC fiona needs it to build.

Reply:
Correct. This is only needed because cuproj uses cuspatial in one of its pytests. And actually we only do that as a demonstration of passing any __cuda_array_interface__ compliant data to cuProj. So if there's a lighter dependency way to test that I'm up for suggestions. I don't want to just use cupy because we use it inside cuproj so it would be just another demonstration of ingesting a cupy array rather than __cuda_array_interface__.

And actually, in the future we may want to test other interactions with cuspatial so I am inclined to just leave this.